src/controllers/attachment.js

```diff
@@ -40,9 +40,9 @@ const updateAttachment = async (req, res) => {
   const { id: currentUser } = req.user
   const updateData = req.body
 
-  await attachmentService.updateAttachment(id, currentUser, updateData)
+  const attachment = await attachmentService.updateAttachment(id, currentUser, updateData)
 
-  res.status(204).end()
+  res.status(200).json({ attachment }).end()
```

Comment:
Why do we need to return attachment as an object if in line 43 we already get an object? Couldn't we just return `attachment` as it is, like the `createAttachments` function?
```suggestion
  res.status(200).json(attachment).end()
```

Reply:
this suggested changes is a genius thing! Solved